Running tests on a test flinger device
Loading environment variables
Using environment file: tf_snapd_db
Deleting test flinger data
Creating job for snapd using a device
Running testflinger client
Submitting job to testflinger
JOB_ID: b66f4a72-4f46-4abe-a5cc-59e1d5aaed4d
Print job: /var/snap/testflinger-cli/current/job.yaml 
job_queue: dragonboard
global_timeout: 36000
provision_data:
    channel: beta
test_data:
    test_cmds: |
        #!/bin/bash
        sudo apt update || ps aux | grep apt
        sudo apt install -y git curl sshpass
        git clone https://github.com/sergiocazzolato/snappy-qa-jobs.git
        (cd snappy-qa-jobs && git checkout master)
        git clone https://git.launchpad.net/snapd-vendor snapd
        (cd snapd && git checkout master)
        . snapd/tests/lib/external/prepare-ssh.sh "{device_ip}" "22" "ubuntu"
        . snappy-qa-jobs/scripts/utils/register_device.sh "{device_ip}" "22" "test" "ubuntu" "sergiocazzolato@yahoo.com.ar"
        . snappy-qa-jobs/scripts/utils/refresh_core.sh "{device_ip}" "22" "test" "ubuntu" "beta" "edge"
        . snappy-qa-jobs/scripts/utils/run_setup.sh "{device_ip}" "22" "test" "ubuntu" ""
        . snappy-qa-jobs/scripts/utils/get_spread.sh
        . snappy-qa-jobs/scripts/utils/run_spread.sh "{device_ip}" "22" "snapd" "external:ubuntu-core-16-arm-64:tests/" "" "tests/main/auto-refresh,tests/main/failover,tests/main/ubuntu-core-services"
Showing job data
This job is currently waiting on a node to become available. You will see no further output until the job starts running.
*******************************************

* Starting setup phase on dragonboard-003 *

*******************************************

Cleaning up container if it exists...
Error response from daemon: No such container: dragonboard003

***********************************************

* Starting provision phase on dragonboard-003 *

***********************************************
Install python-logstash if you want to use logstash logging
2017-10-02 01:27:02,802 dragonboard_003 INFO: BEGIN provision
2017-10-02 01:27:02,802 dragonboard_003 INFO: Booting Master Image
2017-10-02 01:27:02,802 dragonboard_003 INFO: Making sure the master image is booted
2017-10-02 01:27:02,802 dragonboard_003 INFO: Checking if test image booted.
2017-10-02 01:27:03,841 dragonboard_003 INFO: Running ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ubuntu@10.101.51.13 sudo sgdisk -o /dev/mmcblk1
Warning: Permanently added '10.101.51.13' (ECDSA) to the list of known hosts.
Warning: The kernel is still using the old partition table.
The new table will be used at the next reboot or after you
run partprobe(8) or kpartx(8)
The operation has completed successfully.
2017-10-02 01:27:05,821 dragonboard_003 INFO: Running ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ubuntu@10.101.51.13 sudo sync
Warning: Permanently added '10.101.51.13' (ECDSA) to the list of known hosts.
2017-10-02 01:27:06,644 dragonboard_003 INFO: Running snmpset -c private -v1 10.101.48.13 .1.3.6.1.4.1.318.1.1.12.3.3.1.1.4.3 i 2
iso.3.6.1.4.1.318.1.1.12.3.3.1.1.4.3 = INTEGER: 2
2017-10-02 01:27:06,762 dragonboard_003 INFO: Running sleep 10
2017-10-02 01:27:16,789 dragonboard_003 INFO: Running snmpset -c private -v1 10.101.48.13 .1.3.6.1.4.1.318.1.1.12.3.3.1.1.4.3 i 1
iso.3.6.1.4.1.318.1.1.12.3.3.1.1.4.3 = INTEGER: 1
2017-10-02 01:27:26,908 dragonboard_003 INFO: Checking if master image booted.
2017-10-02 01:29:10,832 dragonboard_003 INFO: Flashing Test Image
Warning: Permanently added '10.101.51.13' (ECDSA) to the list of known hosts.
umount: /dev/mmcblk1: not mounted
2017-10-02 01:29:11,351 dragonboard_003 INFO: Running: ['ssh', '-o', 'StrictHostKeyChecking=no', '-o', 'UserKnownHostsFile=/dev/null', 'linaro@10.101.51.13', 'nc 10.101.50.22 59910| gunzip| sudo dd of=/dev/mmcblk1 bs=16M']
Warning: Permanently added '10.101.51.13' (ECDSA) to the list of known hosts.
0+12902 records in
0+12902 records out
747140096 bytes (747 MB) copied, 87.096 s, 8.6 MB/s
Warning: Permanently added '10.101.51.13' (ECDSA) to the list of known hosts.
Warning: Permanently added '10.101.51.13' (ECDSA) to the list of known hosts.

/dev/mmcblk1:
 re-reading partition table
2017-10-02 01:30:39,357 dragonboard_003 INFO: Creating Test User
Warning: Permanently added '10.101.51.13' (ECDSA) to the list of known hosts.
Warning: Permanently added '10.101.51.13' (ECDSA) to the list of known hosts.
Warning: Permanently added '10.101.51.13' (ECDSA) to the list of known hosts.
2017-10-02 01:30:40,414 dragonboard_003 INFO: Booting Test Image
2017-10-02 01:30:40,415 dragonboard_003 INFO: Booting the test image
Warning: Permanently added '10.101.51.13' (ECDSA) to the list of known hosts.
Connection to 10.101.51.13 closed by remote host.
/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/ubuntu/.ssh/id_rsa.pub"

/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: ERROR: ssh: connect to host 10.101.51.13 port 22: No route to host

2017-10-02 01:36:45,219 dragonboard_003 ERROR: exception
Traceback (most recent call last):
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 248, in eat
    return self._dispatch()
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 298, in _dispatch
    result = ingredient.dispatch(self.context)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 152, in dispatch
    return self._dispatch(context, 0)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 179, in _dispatch
    return self._dispatch_None(context, level, retval, command)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 208, in _dispatch_None
    return self._dispatch(context, level + 1)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 179, in _dispatch
    return self._dispatch_None(context, level, retval, command)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 208, in _dispatch_None
    return self._dispatch(context, level + 1)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 165, in _dispatch
    retval = command.invoked(context)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/__init__.py", line 37, in wrapper
    return func(*args, **kwargs)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/__init__.py", line 44, in invoked
    device.provision()
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/dragonboard.py", line 350, in provision
    self.ensure_test_image(test_username, test_password)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/dragonboard.py", line 124, in ensure_test_image
    raise ProvisioningError("Failed to boot test image!")
devices.ProvisioningError: Failed to boot test image!
Traceback (most recent call last):
  File "/srv/testflinger-agent/dragonboard003/env/bin/snappy-device-agent", line 4, in <module>
    __import__('pkg_resources').run_script('snappy-device-agents==0.0.1', 'snappy-device-agent')
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/pkg_resources/__init__.py", line 719, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1504, in run_script
    exec(code, namespace, namespace)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/EGG-INFO/scripts/snappy-device-agent", line 25, in <module>
    main()
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/snappy_device_agents/cmd.py", line 75, in main
    Agent().main()
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/snappy_device_agents/cmd.py", line 71, in main
    return AgentCommandRecipe(self).main(argv, exit)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/recipes/__init__.py", line 119, in main
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 254, in eat
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 310, in _dispatch_failed
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 248, in eat
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 298, in _dispatch
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 152, in dispatch
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 179, in _dispatch
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 208, in _dispatch_None
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 179, in _dispatch
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 208, in _dispatch_None
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 165, in _dispatch
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/__init__.py", line 37, in wrapper
    return func(*args, **kwargs)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/__init__.py", line 44, in invoked
    device.provision()
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/dragonboard.py", line 350, in provision
    self.ensure_test_image(test_username, test_password)
  File "/srv/testflinger-agent/dragonboard003/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/dragonboard.py", line 124, in ensure_test_image
    raise ProvisioningError("Failed to boot test image!")
devices.ProvisioningError: Failed to boot test image!
complete
